src/tests/test_bookmark_endpoints.py

```diff
@@ -0,0 +1,148 @@
+from http import HTTPStatus
+
+from starlette.testclient import TestClient
+from database.session import DbSession
+
+from tests.testutils.users import logged_in_user, ALICE
+from database.model.bookmark.bookmark import Bookmark
+from database.session import DbSession
+from tests.testutils.users import register_asset
+from datetime import datetime
+from database.model.agent.person import Person
+from database.model.agent.contact import Contact
+from database.authorization import User
+
+def test_create_bookmark(
+    client: TestClient,
+    person: Person) -> None:
+
+    register_asset(person)
+    response = client.get("/persons")
+    identifier = response.json()[0]["identifier"]
```

Comment:
```suggestion
    identifier = register_asset(person)
```
The identifier is immediately returned by the `register_asset` function, so we can avoid a round trip through `client`.